vlasiator.cpp

```diff
@@ -647,6 +647,9 @@ int main(int argn,char* args[]) {
       );
    }
 
+   // computing coupling of grids (recalculated when load balancing)
+   computeCoupling(mpiGrid, cells, momentsGrid, onDccrgMapRemoteProcessGlobal, onFsgridMapRemoteProcessGlobal, onFsgridMapCellsGlobal);
+
```

Comment:
Also here, this call could just be put inside balanceLoad(). (see comment further down)